public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -138,6 +154,18 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
     scroll(0, 0);
   }, []);
 
+  // Getting all visible indices. Will re-fetch if changes to the detector (e.g.,
+  // detector starts, result index recreated or user switches tabs to re-fetch detector)
+  useEffect(() => {
+    const getInitialIndices = async () => {
+      await dispatch(getIndices('')).catch((error: any) => {
+        console.error(error);
+        core.notifications.toasts.addDanger('Error getting all indices');
```

Comment:
If user has no permission to list indices, what error message will show up?

Reply:
It would be whatever is returned from the backend, like the regular permissions errors. The reason we only have a generic message in the toast is tor prevent some verbose / not formatted message show up which could add confusion. But it's logged on client-side console as well as server-side (and since it's permissions-related, would be logged in backend also)

Reply:
Cool, it's ok to show regular permissions errors.